exonum-java-binding/core/rust/ejb-app/start_qa_node.sh

```diff
@@ -34,15 +34,20 @@ echo "CURRENT_DIR=${EJB_APP_DIR}"
 EJB_ROOT=$(realpath "../../..")
 echo "PROJ_ROOT=${EJB_ROOT}"
 
+# Find the artifact
+ARTIFACT_PATH="$(find ${EJB_ROOT} -type f -name exonum-java-binding-qa-service-*-artifact.jar)"
+echo "ARTIFACT_PATH=${ARTIFACT_PATH}"
+
+# Prepare the services configuration file
+SERVICES_CONFIG_FILE="ejb_app_services.toml"
+SERVICE_NAME="ejb-qa-service"
+echo "[user_services]" > ${SERVICES_CONFIG_FILE}
+echo "${SERVICE_NAME} = '${ARTIFACT_PATH}'" >> ${SERVICES_CONFIG_FILE}
```

Comment:
I think adding special parameter to the Run configuration would be better than a file in working directory, won't it?

Reply:
Which parameter? If the list of services, then, IIRC, it is needed _before_ arguments are processed by the framework.

Reply:
Ah, indeed, always forget about that fact

Reply:
This file is a temporary configuration feature that will get replaced by (?) some transactions.